README.md

```diff
@@ -0,0 +1,108 @@
+![](https://img.shields.io/badge/brainconnect93-black?labelColor=green)
+
+# Budget-App
+
+> This is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what
+
+## Built With
+
+- Ruby on Rails [click here >>](https://guides.rubyonrails.org/)
+- Rubocop: [Rubocop](https://rubocop.org/)
+- UML schema diagram
+- Bootstrap
+- Ruby
+
+### Live demo
+
+[Live Demo Link]("https://brainconnect-budget-app.herokuapp.com")
+
+[Loom video]("https://www.loom.com/share/b8855a40899d48569858dd2d71461eb8")
```

Comment:
- [ ] very good presentation of your project, however, the video should contain a highlight about a piece of the code you find interesting or are proud of making.

Reply:
@mikethreels thanks for your feed back, i will work on that right now